packages/quantic/cypress/e2e/default-2/search-box/search-box.cypress.ts

```diff
@@ -0,0 +1,526 @@
+              Actions.pressEnterOnSearchBox(textarea);
+              const clickedSuggestionIndex = 0;
+              Expect.searchWithQuery(
+                exampleRecentQueries[clickedSuggestionIndex],
+                {
+                  LSkey: recentQueriesLSKey,
+                  queries: exampleRecentQueries,
+                }
+              );
+              Expect.logClickRecentQueries(
+                exampleRecentQueries[clickedSuggestionIndex]
+              );
+            });
+          });
+        });
+
+        describe('when using the keyboard to select a query suggestion', () => {
+          const exampleRecentQueries = ['foo', 'bar'];
+
+          beforeEach(() => {
+            setRecentQueriesInLocalStorage(exampleRecentQueries);
+            visitSearchBox({...defaultOptions, textarea});
+            mockQuerySuggestions(exampleQuerySuggestions);
+          });
+
+          it('should properly navigate and select the right suggestion', () => {
+            const expectedQuerySuggestions = [
+              ...exampleRecentQueries,
+              ...exampleQuerySuggestions,
+            ];
+            scope('when loading standalone search box', () => {
+              Expect.displayInputSearchBox(true, textarea);
+              Expect.displaySearchButton(true);
+            });
+
+            scope('when focusing on the search box input', () => {
+              Actions.focusSearchBox(textarea);
+              cy.wait(InterceptAliases.QuerySuggestions);
+
+              Expect.displaySuggestionList(true);
+              Expect.displayClearRecentQueriesButton(true);
+              Expect.numberOfQuerySuggestions(expectedQuerySuggestions.length);
+              Expect.querySuggestionsEquals(expectedQuerySuggestions);
+            });
+
+            scope('when selecting a recent query with the keyboard', () => {
+              // First arrow down press to go to the clear recent queries option.
+              Actions.pressDownArrowOnSearchBox(textarea);
+              // pressing the arrow down button to surpass every recent query option.
+              for (let i = 0; i < exampleRecentQueries.length + 1; i++) {
+                Actions.pressDownArrowOnSearchBox(textarea);
+              }
+
+              Actions.pressEnterOnSearchBox(textarea);
+              const clickedSuggestionIndex = 0;
+              Expect.searchWithQuery(
+                exampleQuerySuggestions[clickedSuggestionIndex],
+                {
+                  LSkey: recentQueriesLSKey,
+                  queries: [
+                    exampleQuerySuggestions[clickedSuggestionIndex],
+                    ...exampleRecentQueries,
+                  ],
+                }
+              );
+              Expect.logClickSuggestion(
+                exampleQuerySuggestions[clickedSuggestionIndex]
+              );
+            });
+          });
+        });
+
+        describe('when a custom value is set for the property numberOfSuggestions', () => {
```

Comment:
```suggestion
        describe('the property #numberOfSuggestions', () => {
```
Are we not doing this kind of pattern anymore?

Reply:
It's a good pattern  I will update